src/pages/Integrations/Create/helpers.js

```diff
@@ -0,0 +1,31 @@
+// eslint-disable-next-line react/display-name
+export const iconMapper = (sourceTypes) => (name) => {
```

Comment:
should we maybe change this to 'integrationTypes' ?

Reply:
yeah that's a good idea